docs/development_setup.md

```diff
@@ -47,8 +47,9 @@ A minimal database setup for running tests is provided. If the build argument `O
 
 ### Requirements
 
-- Python 3.8+
-- Poetry 1.1.0+
+- Python: ~3.8
+- R: ~4.0
+- Poetry: ~1.0
```

Comment:
An approximate version isn't helpful. We should have a minimum version or version range.

Reply:
I used those notation because they are used by poetry. But I recognize that this is confusing. I changed it to use `<` and `>`.